src/whitelist.hpp

```diff
@@ -0,0 +1,62 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (C) 2022, KNS Group LLC (YADRO).
+ */
+#pragma once
+
+#include <filesystem>
+#include <set>
+
+namespace fs = std::filesystem;
+
+namespace fssync
+{
+
+namespace details
+{
+struct Pred
```

Comment:
What does `Pred` stand for? Did you mean `Prefix` ?

Reply:
I meant `Predicate`. Renamed to `PathsComparer`.